### PR TITLE
feat: Provide more token lists options

### PR DESCRIPTION
**Pull request by ChefMomota**

### CODE REVIEW

File: src/config/constants/lists.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
 const PANCAKE_EXTENDED = 'https://tokens.pancakeswap.finance/pancakeswap-extended.json'
 const PANCAKE_TOP100 = 'https://tokens.pancakeswap.finance/pancakeswap-top-100.json'
+const COINGECKO = 'https://token-list-av1oidics.pancake.run/coingecko.json'
```
Comment (ChefMomota) [inline src/config/constants/lists.ts:3]: url will be change after pancake-toolkit pr merge.

File: src/state/lists/hooks.ts [hunk 2]
```diff
@@ -71,8 +71,13 @@ export function listToTokenMap(list: TokenList): TokenAddressMap {
             return { ...list.tags[tagId], id: tagId }
           })
           ?.filter((x): x is TagInfo => Boolean(x)) ?? []
+
       const token = new WrappedTokenInfo(tokenInfo, tags)
-      if (tokenMap[token.chainId][token.address] !== undefined) throw Error('Duplicate tokens.')
+      if (tokenMap[token.chainId][token.address] !== undefined) {
+        console.warn(`Duplicate token skipped: ${token.address}`)
```
Comment (chefilip) [inline src/state/lists/hooks.ts:77]: Is this warning useful ?

Comment (ChefMomota) [reply src/state/lists/hooks.ts:77]: If using throw Error, when has duplicate tokens, in our FE will keep calling and website will get stuck.

File: src/state/user/hooks/index.tsx [hunk 3]
```diff
@@ -418,6 +418,9 @@ export function toV2LiquidityToken([tokenA, tokenB]: [Token, Token]): Token {
 export function useTrackedTokenPairs(): [Token, Token][] {
   const { chainId } = useActiveWeb3React()
   const tokens = useAllTokens(true)
+  const filterTokens = Object.keys(tokens)
+    .slice(0, 1000)
```
Comment (chef-jojo) [inline src/state/user/hooks/index.tsx:422]: Note that this is intermediate solution, we will fetch too many pairs if token list is too large

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**